kayenta-judge/src/main/scala/com/netflix/kayenta/judge/classifiers/metric/BaseMetricClassifier.scala

```diff
@@ -56,6 +56,20 @@ object NaNStrategy {
   }
 }
 
+sealed trait OutlierStrategy
+object OutlierStrategy {
+  case object Remove extends OutlierStrategy
+  case object Ignore extends OutlierStrategy
+
+  def parse(outlierStrategy: String): OutlierStrategy = {
+    outlierStrategy match {
+      case "remove" => OutlierStrategy.Remove
+      case "ignore" => OutlierStrategy.Ignore
```

Comment:
This strategy is a little odd from a phrasing perspective. To me, if the outlier strategy was “ignore” it would remove outliers, i.e., we want to ignore outliers in the analysis. An alternative is that we could call this strategy "keep".

Reply:
"keep" works for me.

Reply:
Done.